data/wi/legislature/Rob-Hutton-6d7721b2-fc33-44b4-b9de-e2acb1e6d3de.yml

```diff
@@ -0,0 +1,32 @@
+id: ocd-person/6d7721b2-fc33-44b4-b9de-e2acb1e6d3de
+name: Rob Hutton
+given_name: Rob
+family_name: Hutton
+email: Sen.Hutton@legis.wisconsin.gov
+image: https://docs.legis.wisconsin.gov/2023/legislators/senate/2560.jpg
+party:
+- name: Republican
```

Comment:
@bfossen-ce this person was in the Wisconsin assembly district 13 from 2012 until January 3 2021. https://ballotpedia.org/Rob_Hutton